src/components/DashboardPage/index.jsx

```diff
@@ -32,7 +30,7 @@ const DashboardPage = ({ classes, children, title, subtitle }) => (
         <small className={classes.subtitle}>{subtitle}</small>
       </h1>
     </div>
-    {children}
+    {children.map((child, index) => <div key={index}>{child}</div>)}
```

Comment:
Instead of wrapping in `div`s, you should be able to wrap this in a Fragment:

```jsx
import { Fragment } from 'react';

// ...

<Fragment>{children}</Fragment>
```